wagtail/wagtailadmin/wagtail_hooks.py

```diff
@@ -30,3 +31,29 @@ def register_settings_menu():
 @hooks.register('register_permissions')
 def register_permissions():
     return Permission.objects.filter(content_type__app_label='wagtailadmin', codename='access_admin')
+
+
+@hooks.register('register_page_listing_buttons')
+def page_listing_buttons(page, page_perms, is_parent=False):
```

Comment:
It might be nice if the hook didn't have to care about where/how the buttons are displayed so third party apps wouldn't have to be updated if we ever decide to change where to display the buttons.